tests/lib/prepare.sh

```diff
@@ -44,9 +44,9 @@ disable_refreshes() {
 
 setup_systemd_snapd_overrides() {
     START_LIMIT_INTERVAL="StartLimitInterval=0"
-    if [[ "$SPREAD_SYSTEM" = opensuse-42.2-* ]]; then
+    if [[ "$SPREAD_SYSTEM" =~ opensuse-42.[23]-* ]]; then
         # StartLimitInterval is not supported by the systemd version
```

Comment:
If the problem is the systemd version, why are we not checking it instead of having to keep a manual mapping? What is the actual systemd version that fails, and when was it fixed? @sergiocazzolato Do you know?

Reply:
Thanks, that's a fair point, I'm happy to revisit this soon if we can find answer to this question re systemd version. In the meantime merging this to unblock our testing.